src/index.css

```diff
@@ -1,4 +1,8 @@
-body {
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/* body {
   margin: 0;
   font-family:
     -apple-system,
```

Comment:
Please remove unused pieces of code from your codebase in order to make your codebase clean